<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>SelfTestWidget</class>
    <widget class="QWidget" name="SelfTestWidget">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>722</width>
                <height>421</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <widget class="QStackedWidget" name="stackedWidget">
                    <property name="currentIndex">
                        <number>0</number>
                    </property>
                    <widget class="QWidget" name="test_page">
                        <layout class="QGridLayout" name="gridLayout">
                            <item row="2" column="3">
                                <widget class="QPushButton" name="next_button">
                                    <property name="text">
                                        <string>Nächste Frage</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="0" column="0" colspan="4">
                                <widget class="QLabel" name="question_label_test">
                                    <property name="text">
                                        <string>Ein angreifender Spieler wird von einem Verteidiger genau auf der
                                            Strafraumlinie durch Beinstellen zu Fall gebracht. Welche Entscheidung muss
                                            der SR treffen?
                                        </string>
                                    </property>
                                    <property name="wordWrap">
                                        <bool>true</bool>
                                    </property>
                                    <property name="margin">
                                        <number>20</number>
                                    </property>
                                </widget>
                            </item>
                            <item row="2" column="1" colspan="2">
                                <widget class="QPushButton" name="switch_eval_button">
                                    <property name="text">
                                        <string>Evaluieren</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="2" column="0">
                                <widget class="QPushButton" name="previous_button">
                                    <property name="text">
                                        <string>Vorherige Frage</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="1" column="0" colspan="4">
                                <widget class="QTextEdit" name="user_answer_test">
                                    <property name="html">
                                        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
                                            &lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot;
                                            /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
                                            p, li { white-space: pre-wrap; }
                                            &lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI';
                                            font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
                                            &lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px;
                                            margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Strafstoß,
                                            Linie gehört zum Strafraum&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                                        </string>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </widget>
                    <widget class="QWidget" name="eval_widget">
                        <layout class="QGridLayout" name="gridLayout_2">
                            <item row="2" column="3">
                                <widget class="QWidget" name="eval_button_widget" native="true">
                                    <layout class="QVBoxLayout" name="verticalLayout_2">
                                        <item>
                                            <widget class="QPushButton" name="incorrect_button">
                                                <property name="text">
                                                    <string>Falsch</string>
                                                </property>
                                            </widget>
                                        </item>
                                        <item>
                                            <widget class="QPushButton" name="correct_button">
                                                <property name="text">
                                                    <string>Richtig</string>
                                                </property>
                                            </widget>
                                        </item>
                                    </layout>
                                </widget>
                            </item>
                            <item row="2" column="2">
                                <widget class="QTextEdit" name="correct_answer_eval">
                                    <property name="html">
                                        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
                                            &lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot;
                                            /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
                                            p, li { white-space: pre-wrap; }
                                            &lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI';
                                            font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
                                            &lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px;
                                            margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Der SR muss
                                            auf Strafstoß entscheiden. Die Begrenzungslinien gehören zu dem Raum, den
                                            sie umschließen. (Hier: Strafraumlinie zum Strafraum)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                                        </string>
                                    </property>
                                    <property name="textInteractionFlags">
                                        <set>Qt::NoTextInteraction</set>
                                    </property>
                                </widget>
                            </item>
                            <item row="2" column="0" colspan="2">
                                <widget class="QTextEdit" name="user_answer_eval">
                                    <property name="html">
                                        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
                                            &lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot;
                                            /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
                                            p, li { white-space: pre-wrap; }
                                            &lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI';
                                            font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
                                            &lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px;
                                            margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Strafstoß,
                                            Linie gehört zum Strafraum&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                                        </string>
                                    </property>
                                    <property name="textInteractionFlags">
                                        <set>Qt::NoTextInteraction</set>
                                    </property>
                                </widget>
                            </item>
                            <item row="0" column="0" colspan="4">
                                <widget class="QLabel" name="question_label_eval">
                                    <property name="text">
                                        <string>Ein angreifender Spieler wird von einem Verteidiger genau auf der
                                            Strafraumlinie durch Beinstellen zu Fall gebracht. Welche Entscheidung muss
                                            der SR treffen?
                                        </string>
                                    </property>
                                    <property name="wordWrap">
                                        <bool>true</bool>
                                    </property>
                                    <property name="margin">
                                        <number>20</number>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </widget>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
